sql/moz-fx-data-shared-prod/telemetry_derived/clients_histogram_aggregates_v2/query.sql

```diff
@@ -109,7 +111,9 @@ clients_histogram_aggregates_partition AS (
   SELECT
     *
   FROM
-    telemetry_derived.clients_histogram_aggregates_v2
+    `moz-fx-data-shared-prod.telemetry_derived.clients_histogram_aggregates_v2`
+  WHERE
+    submission_date = DATE_SUB(DATE(@submission_date), INTERVAL 1 DAY)
```

Comment:
I added this parameter to be extra safe. The table is overwritten at every execution so there should only be one submission date, but just in case there are more...